src/lib/Components/OpaqueImageView.tsx

```diff
@@ -81,6 +83,9 @@ export default class OpaqueImageView extends React.Component<Props, State> {
 
   imageURL() {
     const imageURL = this.props.imageURL
+    if (this.props.skipGemini) {
```

Comment:
@l2succes so here, we just serve the URL right back, as opposed to trying to process on-the-fly.

It's just a temp hack for now just to get something working.